### PR TITLE
108 use nunjucks templates

### LINKED ISSUE #108: Integrate Nunjucks templates into JS tests
### Vision

We want to use html templates as DOM fixtures for our JS tests
### Current State

We are hard coding HTML into the the tests. We added templates but we would get errors finding imported templates.
### Anti-Vision

What is out of scope or goes against the original intention?

### PR DESCRIPTION
**Pull request by misscs**
### Issues

Allow useage of Nunjucks templates in our JS test files. Resolves #108 
### Impacted Areas
- Accordion module tests
- @todo nightshade
- @todo storefront static
### Steps to Reproduce

Run `npm run test`. All 3 tests should pass.

### CODE REVIEW

**Comment by misscs** [comment]
@stewartyu This PR let's us use Nunjucks templates in our test files. It is built on top of the Ava test, but should easily translate to Mocha/Chai tests. The solution is in 97995627e4979b0e. 

tldr:
- Need to use `__dirname` 
- Filesystem loader needs to be absolute. 
- Use relative path for calls to render

**Comment by stewartyu** [comment]
🎉 Awesome!
